pom.xml

```diff
@@ -33,6 +33,10 @@
 
     <modules>
         <module>apm-protocol</module>
+        <module>oap-server</module>
+        <module>oap-server-bom</module>
+        <module>apm-webapp</module>
+        <module>apm-dist</module>
```

Comment:
This is the reason we could close `activeByDefault` for those modules, right?

Reply:
> This is the reason we could close `activeByDefault` for those modules, right?

Yes, and because these modules are added in the previous default profiles, so I think this is OK we just make it default here